main_test.go

```diff
@@ -33,9 +33,9 @@ func TestBuildTable(t *testing.T) {
 		head := &CoverProfile{}
 
 		assert.Equal(t, strings.TrimLeft(`
-package                                                                            before    after    delta
--------                                                                            ------    -----    -----
-                                                                          total:        -        -      n/a
+|package                                                                         |  before|   after|   delta|
+|-------                                                                         |  ------|   -----|   -----|
```

Comment:
I'd also like to see left aligned `package` and right aligned everything else.

Sorry, this is the test case, but you get the idea of what I think it should be 

E.g.

```suggestion
| package                                                                        | before |  after |  delta |
|:---                                                                            |    ---:|    ---:|    ---:|
```

Basically use of `:` colons to set column alignment.